desktop/core/src/desktop/conf.py

```diff
@@ -1818,6 +1818,13 @@ def get_instrumentation_default():
   default=False
 )
 
+ENABLE_NEW_FILE_UPLOADER = Config(
+  key="enable_new_file_uploader",
+  help=_("Feature flag to enable new Hue file uploader."),
```

Comment:
Nit: Maybe not required to mention everywhere that config is a flag?

Reply:
changed it.